src/sources/soundsourceproxy.cpp

```diff
@@ -609,11 +656,11 @@ bool SoundSourceProxy::updateTrackFromSource(
     }
 
     // Parse the tags stored in the audio file
-    auto metadataImportedFromSource =
+    auto [metadataImportResult, sourceSynchronizedAt] =
```

Comment:
Nice workaround for the pair.
 
Can we rename to sourceSynchronizedAt to something that reveals that it is a QDateTime?

Reply:
I disagree. The name does not need to repeat the type in a strongly types language.

Reply:
The issue here is that the type is out of sight and the variable is IMHO badly named. The reader has no hint what happens here. He has to reference other files to get an idea.

How about adding a brief comment instead?

Reply:
I am not going to respond to "type is out of sight" comments. This change is an improvement to the code before which also didn't repeat the types.

Reply:
If I am not allowed to write reasonably modern C++ code I will stop contributing.

Reply:
https://en.cppreference.com/w/cpp/language/structured_binding

Reply:
As I have written initially I like the solution around the anonymous pair issue. This was serious. I have now also understand why the variable is named as it is. But the original issue of the hard to understand code still applies. I will propose an improved comment, which is probably the best solution.

Reply:
> The issue here is that the type is out of sight

I think using `auto` is okay. It's just a local variable. Your IDE probably shows the type when hovering. And even if it doesn't, the compiler will tell you what type it is when you try to use it.

Reply:
The issue is already fixed by the additional comment. Thank you.